documentation/writing-to-vespa.html

```diff
@@ -248,166 +206,3 @@ <h2 id="performance">Performance</h2>
 </p>
 
 
-<h3 id="load-priority">Load priority</h3>
-<p>
-Read/write operations are assigned default <em>priority levels</em> -
-modify these in <em>services.xml</em> using <a href="reference/services.html#clients">load types</a>.
-Metrics are generated per load type - example:
-<pre>
-&lt;get&gt;
-  &lt;default&gt;
-    &lt;count count="1"/&gt;
-    &lt;latency average="0.125977" last="0.125977" min="0.125977" max="0.125977" count="1"/&gt;
-    &lt;not_found count="1"/&gt;
-  &lt;/default&gt;
-  &lt;foo&gt;
-    &lt;count count="1"/&gt;
-    &lt;latency average="0.25708" last="0.25708" min="0.25708" max="0.25708" count="1"/&gt;
-  &lt;/foo&gt;
-  &lt;sum&gt;
-    &lt;count count="2"/&gt;
-    &lt;latency average="0.191528" last="0.25708" min="0.125977" max="0.25708" count="2"/&gt;
-    &lt;not_found count="1"/&gt;
-  &lt;/sum&gt;
-&lt;/get&gt;
-</pre>
-</p>
-
-
-<h2 id="combine-values-in-indexing-statements">Combine values in indexing statements</h2>
-<p>
-Vespa allows flexibility in how documents are turned into index structure
-data through <em>indexing statements</em>. Using indexing statements, multiple
-document fields can be used to produce one index structure field. For example,
-the index statement
-<pre>
-input field1 . input field2 | attribute field2;
-</pre>
-will combine the document fields <em>field1</em> and <em>field2</em>
-into the attribute named <em>field2</em>.
-When partially updating document which contains indexing statement which
-combines multiple fields the following rules apply:
-<ul>
-  <li>Only attributes where <em>all</em> the source values are available in
-    the source document update will be updated</li>
-  <li>The document update will fail when indexed (only) if <em>no</em>
-    attributes end up being updated when applying the rule above</li>
-</ul>
-</p><p>
-Example: If a search definition has the indexing statements
-<pre>
-input field1 | attribute field1;
-input field1 . input field2 | attribute field2;
-</pre>
-the following will happen for various partial updates.
-<table class="table">
-  <thead>
-  <tr><th>Partial update contains</th><th>Result</th></tr>
-  </thead></tbody>
-  <tr><td>field1</td><td>field1 is updated</td></tr>
-  <tr><td>field2</td><td>The update fails</td></tr>
-  <tr><td>field1 and field2</td><td>field1 and field2 is updated</td></tr>
-  </tbody>
-</table>
-</p>
```

Comment:
This section should be moved to search-definitions.html - i will add it there, so ok to remove from here